model/AbstractRdsResultStorage.php

```diff
@@ -622,18 +622,22 @@ protected function serializeVariableValue($value)
     /**
      * @param array $data
      *
+     * @param array $types
      * @throws DuplicateVariableException
      */
-    private function insertMultiple(array $data)
+    private function insertMultiple(array $data, array $types = [])
```

Comment:
Let's use return types now.

Reply:
Why would we ever need to pass the types to this function?

Reply:
the development was done by @siwane , and the goal of that was to be able to specify the type that you want to cast to when interting